src/Illuminate/Database/Connectors/PostgresConnector.php

```diff
@@ -81,34 +81,42 @@ protected function configureTimezone($connection, array $config)
     }
 
     /**
-     * Set the schema on the connection.
+     * Set the search_path on the connection.
      *
      * @param  \PDO  $connection
      * @param  array  $config
      * @return void
      */
-    protected function configureSchema($connection, $config)
+    protected function configureSearchPath($connection, $config)
     {
-        if (isset($config['schema'])) {
-            $schema = $this->formatSchema($config['schema']);
+        if (isset($config['searchpath']) || isset($config['schema'])) {
+            $searchPath = $this->formatSearchPath($config['searchpath'] ?? $config['schema']);
 
-            $connection->prepare("set search_path to {$schema}")->execute();
+            $connection->prepare("set search_path to {$searchPath}")->execute();
         }
     }
 
     /**
-     * Format the schema for the DSN.
+     * Format the search path for the DSN.
      *
-     * @param  array|string  $schema
+     * @param  array|string  $searchPath
      * @return string
      */
-    protected function formatSchema($schema)
+    protected function formatSearchPath($searchPath)
     {
-        if (is_array($schema)) {
-            return '"'.implode('", "', $schema).'"';
+        if (is_array($searchPath)) {
```

Comment:
Small suggestion; instead of having `is_array / else` (like the old code), I _think_ we can get rid of nesting the whole method if you actually convert the string into an array:
```php
$searchPath = (array) $searchPath;
// the code always just has to handle arrays
```

See:
```php
$ php -r 'var_dump( (array) "foo");'
array(1) {
  [0]=>
  string(3) "foo"
}

$ php -r 'var_dump( (array) ["foo"]);'
array(1) {
  [0]=>
  string(3) "foo"
}

Reply:
Precisely. That is one of the things we are seeking to change.